examples/starknet-react-next/src/global.css

```diff
@@ -0,0 +1,7 @@
+@import url("../node_modules/@cartridge/ui-next/dist/themes/default.css");
+@import url("../node_modules/@cartridge/ui-next/dist/themes/dark.css");
+@import url("../node_modules/@cartridge/ui-next/dist/themes/fonts.css");
```

Comment:
ideally, would be nice to import like `@cartridge/ui-next/themes/default.css` but not sure how to make it work.